### DISCUSSION TITLE
Custom Font - WOFF in BIRT Report

### DISCUSSION THREAD

**Discussion by kiranduba** [eclipse-birt/birt#1454] [Q&A]
Hello Everyone,
I'm trying to use class defined in WOFF file, how can I add this external file in my BIRT report. Please help here. Using BIRT 4.8.0

Thanks,
Kiran

**Comment by hvbtup** [comment]
AFAIK BIRT only supports TTF files.

**Reply by kiranduba** [reply]
Hi @hvbtup I converted WOFF file to TTF and installed it in my Windows, while designing BIRT template I can see that font in editor, BUT unfortunately that font is not getting rendered in generated PDF. 
I also added the path of that font file in fontConfig.xml, but no luck. 
Please suggest.
